build/yaml/pipelines/signed.yaml

```diff
@@ -10,6 +10,7 @@ name: $(date:yyyyMMdd)$(rev:rr)
 
 variables:
   TeamName: ClrInstrumentationEngine
+  Packaging.EnableSBOMSigning: true
```

Comment:
Should this just be a variable on the postbuild job instead of all of the jobs in this stage?